### PR TITLE
Fixes braces/brackets in files breaking custom music

### LINKED ISSUE #338: [ BUG REPORT ] -  Custom Music AutoPlay Crash
the autoplay for custom music works, but crashes when it plays a song that seems to exceed a file size compatibility limit (it may be trying to do multiple things at once).

[traceback.txt](https://github.com/Just-Natsuki-Team/NatsukiModDev/files/9035377/traceback.txt)

### PR DESCRIPTION
**Pull request by Blizzardsev**
Closes #338

### CODE REVIEW

File: game/options.rpy [hunk 1]
```diff
@@ -23,7 +23,7 @@ define gui.show_name = False
 
 ## The version of the game.
 
-define config.version = "0.0.2"
+define config.version = "0.0.3"
```
Comment (multimokia) [inline game/options.rpy:26]: We should only change this when we're going to distribute a release, and this should be in its own commit/pr

Comment (Blizzardsev) [reply game/options.rpy:26]: Got it

**Review by multimokia — APPROVED** [review]
